contributing/code/tests.rst

```diff
@@ -32,7 +32,7 @@ tests, such as Doctrine, Twig and Monolog. To do so,
 
     .. code-block:: terminal
 
-        $ COMPOSER_ROOT_VERSION=5.4.x-dev composer update
+        $ COMPOSER_ROOT_VERSION=7.0.x composer update
```

Comment:
```suggestion
        $ COMPOSER_ROOT_VERSION=7.0.x-dev composer update
```

we need to keep the suffix in cases where we reference not yet released packages

Reply:
Alright, i will fix this. I taught that because 7.0 is already released so not needed